frontends/web/src/routes/exchange/btcdirect.tsx

```diff
@@ -97,8 +104,86 @@ export const BTCDirect = ({ accounts, code }: TProps) => {
     }, 200);
   };
 
+  const handlePaymentRequest = useCallback(async (event: MessageEvent) => {
```

Comment:
It's not strictly a (SLIP-24) payment request 😇

Reply:
correct. but it is a payment request. Should I rename the function?

Reply:
I think it makes sense to avoid the term "payment request" to avoid confusion with SLIP-24

Reply:
we can ofc rename it, but for PR I realized this will also need to be changed in the static file. So I'll keep for now.

a possible alternative could be handlePaymentIntend